frontend/src/styles/generic/layout.scss

```diff
@@ -46,6 +46,14 @@ body {
   height: inherit !important;
 }
 
+.nowrap {
+  white-space: nowrap;
+}
+
+.flex {
+  display: flex;
+}
+
```

Comment:
This is the exact same as `.inline-flex`

the only difference being `.inline-flex` has additional browser support with the flex mixin. 

```
@mixin flex() {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
}
```

Reply:
Cool, removed and used inline-flex.